dbschema/models.py

```diff
@@ -48,6 +49,17 @@ class DialogQuestions(Base):
     __tablename__ = 'dialog_questions'
     question_id = Column(Integer, primary_key=True)
     question_description = Column(String)
+    
+    
+class FirstAidKit(Base):
+    __tablename__ = 'first_aid_kit'
+    id = Column(Integer, primary_key=True, autoincrement=True)
```

Comment:
```suggestion
    first_aid_kit_id = Column(Integer, primary_key=True, autoincrement=True)
```
This is a good practice in order not to confuse ids (I also just learned this)